Libplanet/Net/Messages/Message.cs

```diff
@@ -0,0 +1,120 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Libplanet.Crypto;
+using NetMQ;
+
+namespace Libplanet.Net.Messages
+{
+    internal abstract class Message
+    {
+        protected enum MessageType : byte
+        {
+            /// <summary>
+            /// Check message to determine peer is alive.
+            /// </summary>
+            Ping = 0x01,
+
+            /// <summary>
+            /// Reply of `Ping`.
```

Comment:
FYI unfortunately C# doc comments use an ancient XML-based syntax instead of Markdown.  Don't mind this, I will write docs for them later.